cmd/env.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pingcap/tiup/pkg/localdata"
+	"github.com/spf13/cobra"
+)
+
+var envList = []string{
+	localdata.EnvNameHome,
+	localdata.EnvNameTelemetryStatus,
+	localdata.EnvNameTelemetryUUID,
+	localdata.EnvNameSSHPassPrompt,
+	localdata.EnvNameSSHPath,
+	localdata.EnvNameSCPPath,
+}
+
+func newEnvCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "env [name1...N]",
+		Short: "Show the list of system environment variable that related to TiUP",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				args = envList
+			}
+			showEnvList(args...)
+			return nil
+		},
+	}
+
+	return cmd
+}
+
+func showEnvList(names ...string) {
+	for _, name := range names {
+		fmt.Printf("%s=\"%s\"\n", name, os.Getenv(name))
```

Comment:
Prefer to show only value if the user specifies environment variables explicitly.
```shell
➜  devel go env GOPATH
/c/devel/gopath
➜  devel go env GOPATH GOROOT
/c/devel/gopath
/usr/local/go
```

Reply:
Yes, I have noticed the golang's behavior, but I don't know what's the benefit of this. Can you explain this?

Reply:
I think to output the value of an envvar instead of `key=value` may help get the value in the script, and the script doesn't need to split the string.

Reply:
OK